docs/sources/references/transformations.md

```diff
@@ -24,11 +24,11 @@ weight: 7003
 
 # Transformations
 
-Server side transformations are introduced in Infinity 2.0. You can perform basic data manipulation in the server side.
+Server-side transformations are introduced in Infinity 2.0. You can perform basic data manipulation in the server side.
 
-To perform server side transformations, you need to add a query type of **transformations**. Once this added, **this will perform server side transformation over all the previous infinity queries with backend parser**.
+To perform server-side transformations, you need to add a query type of **transformations**. Once this added, **this will perform server side transformation over all the previous infinity queries with backend parser**.
 
-> **Warning**: Infinity Transformations/Server side transformations, are not available neither for non infinity datasources nor infinity queries with non-backend parsers.
+> **Warning**: Infinity Transformations/Server-side transformations are available only for Infinity data sources or Infinity queries with backend parsers.
```

Comment:
I think this is actually a note not a warning but you can replace with "warning" if you disagree

```suggestion
{{< admonition type="note" >}}
Infinity Transformations/Server-side transformations are available only for Infinity data sources or Infinity queries with backend parsers.
{{< /admonition >}}
```